source/Test.h

```diff
@@ -0,0 +1,46 @@
+/* Test.h
+Copyright (c) 2019 by Peter van der Meer
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#ifndef TEST_H_
+#define TEST_H_
+
+#include "DataNode.h"
+#include "TestStep.h"
+#include <string>
+
+
+
+// Class representing a single test.
+class Test {
+public:
+	Test();
+	virtual ~Test();
+	
+	// Status indicators for the test that we selected (if any).
+	static const int STATUS_ACTIVE = 0;
+	static const int STATUS_KNOWN_FAILURE = 1;
+	static const int STATUS_MISSING_FEATURE = 2;
+	
+	virtual std::string Name() const;
```

Comment:
Is this a base class? If so, where are the deriving classes? What would they look like (there should be at least two otherwise it doesn't make sense to have a base and a derived class).

Also, would these deriving classes really be expected to specialize things like the `Name` getter?

Reply:
Yes, initially I expected this to become a base class with multiple derived classes, but that was before I moved out some functionality to TestSteps. I will remove the virtual keywords, since I don't expect to derive any classes from this class anymore.